Cargo.toml

```diff
@@ -10,6 +10,7 @@ documentation = "https://docs.rs/xtask-wasm"
 
 [dependencies]
```

Comment:
Use full version qualifiers in our dependencies.

As a library it makes more sense to keep the exact version in Cargo.toml even if it does not impact how the thing works. This is because the Cargo.lock file is usually not versionned nor shipped for libraries, and if a dependency breaks something we need to know on which version it worked last.

`anyhow = "1.0"` => `anyhow = "1.0.52"`